sys/linux/seccomp.txt

```diff
@@ -46,8 +47,18 @@ seccomp_notif_resp {
 	flags	const[0, int32]
 }
 
+seccomp_notif_addfd {
+	id		seccomp_id
+	flags		flags[addfd_flags, int32]
+	srcfd		fd
+	newfd		int32
+	newfd_flags	flags[newfd_flags, int32]
+}
+
 seccomp_flags = 0, SECCOMP_FILTER_FLAG_TSYNC, SECCOMP_FILTER_FLAG_LOG, SECCOMP_FILTER_FLAG_SPEC_ALLOW
 seccomp_flags_listener = SECCOMP_FILTER_FLAG_NEW_LISTENER, SECCOMP_FILTER_FLAG_LOG_LISTENER, SECCOMP_FILTER_FLAG_SPEC_ALLOW_LISTENER
+newfd_flags = O_CLOEXEC
+addfd_flags = SECCOMP_ADDFD_FLAG_SETFD
```

Comment:
These should probably be called `seccomp_newfd_flags` and `seeccomp_addfd_flags`, since all the types go into a global namespace.

Reply:
Fixed. I changed the second one to seccomp_addfd_newfd_flags since that's probably more appropriate.